src/inputs/ChipSelectField.tsx

```diff
@@ -59,7 +62,7 @@ export function ChipSelectField<O, V extends Value>(
   const typeScale = fieldProps?.typeScale ?? "sm";
   const isDisabled = !!disabled;
   const showClearButton = !disabled && clearable && !!value;
-  const chipStyles = Css[typeScale].tl.bgGray300.gray900.br16.pxPx(10).pyPx(2).$;
+  const chipStyles = useMemo(() => Css[typeScale].tl.bgGray300.gray900.br16.pxPx(10).pyPx(2).$, [typeScale]);
```

Comment:
If it's too late that's fine, but if it's easy to do, I'd be interested in the before/after profile of just this one change. Doing `Css...$` everytime imo has a chance at actually being cheaper than `useMemo` adding the slight overhead of another hook to track.

Reply:
Not much of a difference, but here's the result. Granted, the perf tests locally for this jump between 6ms and 7ms.

# Without Memo
![Without Memo](https://user-images.githubusercontent.com/1143861/145256293-67450101-60c0-4f24-a645-636cbd643861.png)

# With Memo
![With Memo](https://user-images.githubusercontent.com/1143861/145256297-154f8010-7f23-47ab-9059-077476782fde.png)